### ISSUE TITLE
Generating linux wheels from tag makes incorrect version information

### ISSUE THREAD

**Issue by aveach** [ParallelSSH/ssh2-python#79]
I'm running into:
`ImportError: /usr/local/lib/python3.6/site-packages/ssh2/session.cpython-36m-arm-linux-gnueabihf.so: undefined symbol: libssh2_session_set_last_error` when using the standard 0.17.0 install from pip so decided to install from source instead. 

When I install from source using: https://github.com/ParallelSSH/ssh2-python/archive/0.17.0.post3.tar.gz the version comes up as 0+unknown. This doesn't play well with my requirements.txt files or proper versioning in general. 
```
$ python3.6 -m pip list | grep ssh2
Package       Version  
------------- ---------
ssh2-python   0+unknown
```

**Comment by pkittenis** [comment]
Installing from pip is building from source regardless as you are on an architecture with no binary wheels (arm). The symbol error is because of a conflict with the system installed libssh2, which is fixed in master branch but not released to PyPi due to an issue generating linux wheels.

Github archive is not a source distribution, it's a copy of the repo at the time the tag was taken without git info. It will not have correct versioning as a result. Checkout the tag after cloning repo and `pip install .` instead.

Will leave this open to track fixing wheel generation, so can publish the latest source.

**Comment by aveach** [comment]
Any idea on the eta for updated wheels?

**Comment by pkittenis** [comment]
Have published `0.18.0.post1` with latest changes and fixed linux wheels. Source builds from PyPi should now work correctly as well with a system libssh2 installed.

Latest release on: https://pypi.org/project/ssh2-python/
